save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -209,9 +208,6 @@ class DockerService(
         "LongMethod",
     )
     private fun prepareImageAndVolumeForExecution(resourcesForExecution: Path, execution: Execution): RunConfiguration<PersistentVolumeId> {
-        // collect test suite names, which were selected by user
-        val saveCliExecFlags = " --include-suites \"${execution.getTestSuiteNames().joinToString(DATABASE_DELIMITER)}\" $TEST_SUITES_DIR_NAME"
```

Comment:
Is `--include-suites` flag not necessary anymore?

Reply:
yes, we found that save-cli gets command from backend with batch tests to be run. So `include-suites` is not required

Reply:
I think this flag might have been useful for test sources with wildcard mode enabled. However, we can't preprocess such suites, because tests get discovered only by the tool itself. I think we should add it to validation of test suite sources, if we are going to add one.

Reply:
we have test suites ids on Execution which is used to detect tests are related to current execution (`test suite ids` -> `test suite` -> `tests`). Orchestrator provide `test suite names` using `test suite ids` from `execution`.
wildcard mode should detect tests in similar way for cases when `test suite names` is provided and `specific tests` and when only `specific tests` are provided

Or I missed something?

Reply:
Yes, wildcard mode should work fine in save-cli, however I don't think we can store tests during preprocessing of such suites. We'll get results after execution only.